distribution/lib/Standard/Table/0.0.0-dev/src/Value_Type.enso

```diff
@@ -484,3 +484,11 @@ type Value_Type
    - otherwise, `Text` is chosen as a fallback and the column is kept as-is
      without parsing.
 type Auto
+
+## The type representing the column type being explicitly selected by the user.
+   Arguments: 
+   - type: The type of columns to select. 
+   - strict: If `True`, only columns with exactly the specified types will be selected. 
+     If `False`, columns with related types will also be selected (i.e. ignore size, precision).
+type By_Type
+  By_Type type:Value_Type strict:Boolean=False
```

Comment:
```suggestion
type By_Type
    By_Type type:Value_Type strict:Boolean=False
```

I think we can remove the `strict` option - I think it add complexity for no value.
We need a doc comment on the constructor please.

Should we also make it `types:Vector Value_Type=[]` assuming that we can do nested multi-selects.
I think that might be cleaner but happy with either way.

Reply:
Nested vectors don't work great in the GUI. Too many +s in my opinion. I think the UX is better without the nesting.

Reply:
Removed strict.